docs/troubleshooting/migrate_cstor_pools.md

```diff
@@ -0,0 +1,229 @@
+
+### OpenEBS setup
+List of resources that are created after provisioning cStor pools and volumes
+
+- To know about cStorpoolcluster
+```sh
+$ Kubectl get cspc -n openebs
+NAME         HEALTHYINSTANCES   PROVISIONEDINSTANCES   DESIREDINSTANCES   AGE
+cstor-cspc   3                  3                      3                  108s
+```
+- To know more details about the cStor pools
+```sh
+$ kubectl get cspi -n openebs
+NAME              HOSTNAME            ALLOCATED  FREE   CAPACITY  READONLY  PROVISIONEDREPLICAS  HEALTHYREPLICAS  TYPE    STATUS  AGE
+cstor-cspc-4tr5   ip-192-168-52-185   98k        9630M  9630098k  false     1                    1                stripe  ONLINE  4m26s
+cstor-cspc-xnxx   ip-192-168-79-76    101k       9630M  9630101k  false     1                    1                stripe  ONLINE  4m25s
+cstor-cspc-zdvk   ip-192-168-29-217   98k        9630M  9630098k  false     1                    1                stripe  ONLINE  4m25s
+```
+- To know more about the cStor volumes resources
+```sh
+$ kubectl get cvc,cv,cvr -n openebs
+NAME                                                                          STATUS   AGE
+cstorvolumeconfig.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826   Bound    7m3s
+
+NAME                                                                                           USED   ALLOCATED   STATUS    AGE
+cstorvolumereplica.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826-cstor-cspc-4tr5   6K     6K          Healthy   7m3s
+cstorvolumereplica.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826-cstor-cspc-xnxx   6K     6K          Healthy   7m3s
+cstorvolumereplica.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826-cstor-cspc-zdvk   6K     6K          Healthy   7m3s
+
+NAME                                                                    STATUS    AGE    CAPACITY
+cstorvolume.cstor.openebs.io/pvc-81746e7a-a29d-423b-a048-76edab0b0826   Healthy   7m3s   5Gi
+```
+Here:
+- CVC refers to cStorVolumeManager(Which is responsible for managing cStor volume resources like target service, target deployment, CV and CVR).
+- CV refers to CStorVolume(Which contains the configuration required for process running inside the cStor target pod container).
+- CVR refers to CStorVolumeReplica(Which contains the configuration required to create zfs replica in cStor pool this replica is responsible for reading/writing application IOs to pool. It follows the naming convention like <pv_name>-<pool_name>).
+
+### Lost Node and underlying disk attached to the ndoe
+Performed node and disk lost action as specified [here](#reproduce-the-node-and-underlying-disk-lost-situtation). Now only two nodes are present in the cluster which means whatever data in that corresponding cStor pool was lost. To reconstruct the data we need to make use of the rebuild feature provided by cStor(i.e reconstructing data from other replicas). Below are details after loosing the node and disk
+```sh
```

Comment:
n_p: 
`...cluster which means whatever data in that corresponding cStor pool was lost` ---->
`...cluster which means all existing cStor pools and hence data is lost for the lost node `